engine/launcher/src/main/scala/org/enso/launcher/Main.scala

```diff
@@ -14,12 +14,18 @@ import org.enso.cli.{
 }
 import org.enso.cli.Opts.implicits._
 import cats.implicits._
+import org.enso.launcher.installation.{
+  DistributionInstaller,
+  DistributionManager
+}
+import org.enso.launcher.installation.DistributionInstaller.BundleAction
 
 object Main {
   private def jsonFlag(showInUsage: Boolean): Opts[Boolean] =
     Opts.flag("json", "Use JSON instead of plain text for output.", showInUsage)
 
-  type Config = Unit
+  case class GlobalConfig()
```

Comment:
why not a case object?

Reply:
Right now it could be, or even stay at `Unit` as it were. But in the future possible global toggles will be added in there, so this will need to be a case class then.